en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -334,6 +334,103 @@ When there are null values in custom expressions and are:
+. Complete the initial value for each variable. The default value is defined under each variable. If the field empty, the value is set to the default value for that data type. You can enter a value manually, select any existing data pill (including variables) from your flow, or you can select the xref:ms_composer_custom_expression_editor.adoc#add-custom-expression-to-an-action[*Custom Expression Editor*] to create a custom expression. 
+
+=== Create a Variable Using the JSON Schema
+
+To create one or more new variables using the *JSON Schema* window: 
+
+. <<access-the-variable-step,Access the *Variable* step.>> 
+. In the *Action* dropdown, click *Create Variable*.
+. In the *Variable* section, click *Create Schema*. 
+. In the *JSON Schema* window, enter the properties for the variable you want to create. Note that: 
+.. Each variable is defined within a `Properties`  object with a `Variable Unique ID`.  The `Variable Unique ID` allows the variable to be tracked within your flow. The `Variable Unique ID` can only contain alphanumeric characters, hyphens, and underscores. When using the Visual Editor, this ID is automatically generated for you.
+Each variable must specify the `Name` and data type. To set the properties for the variable definition, complete the following:
+... `Variable Label (Title)`:  The display label for the variable. A value is required. 
+... `Type` : The data type of the variable. The following xref:ms_composer_custom_expression_editor.adoc#data-types[data types] are supported: `String`, `Number`, and `Boolean`. To define a `Date` or `DateTime` variable, the data type should be set to `String` and the `Format` property should be defined. A value is required. 
+... `Format`:  The qualifier for `String` properties that can specify additional data types not natively supported in JSON. For variables, this must be set when defining `Date` or `DateTime` variables as `date` or `date-time` respectively. A value in this field is optional. 
+
+//this should be a result statement... but how?! The Dev insisted that we have this section but I have no idea how to format it and I really don't even know how to introduce it.
+The properties are entered into the *JSON Schema* window. For example: 
+
+[source,JSON]
+----
+{
+"properties": {
+"var-01": {
+"type": "string",
+"title": "My String"
+},
+"var-02": {
+"type": "number",
+"title": "My Number"
+},
+"var-03": {
+"type": "boolean",
+"title": "My Boolean"
+},
+"var-04": {
+"type": "string",
+"title": "My Date",
+"format": "date"
+},
+"var-05": {
+"type": "string",
+"title": "My Datetime",
+"format": "date-time"
+}
+}
+}
+----
+//Does anyone have any idea how to indent this block and make the step numbers continue instead of restart?! 
+. Optionally, to add additional variables in the JSON Schema, enter them in the window.
+. Click *Apply*. +
+The *Create Variables* window appears, displaying one or more new variables added to the flow. 
```

Comment:
Necessary?

Kim prefers shows.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22appears,%22%22displays%22

Reply:
I think it is necessary. This tells the user what they should see on the screen. I don't think "shows" works in this context.